### DISCUSSION TITLE
Jpeg video as source over HTTP

### DISCUSSION THREAD

**Discussion by acorreane** [bluenviron/mediamtx#1712] [Questions] [answered]
exists a possibility for proxying a jpeg stream to rtsp with ffmpeg encoding?

im trying a path source as https://cameras.santoandre.sp.gov.br/coi02/ID_048
 idea: stream to a network video recorder on local network with mediamtx

**Comment by aler9** [comment, marked as answer]
Hello ingesting Motion-JPEG streams is something that is not currently supported. Nonetheless, i've created a feature request in order to track the development of the feature: #1714

**Comment by acorreane** [comment]
my project worked with 2 appliances, mediamtx serving rtsp and listening rtmp streams from another project remux.io from
https://github.com/remux-io 
its working perfectly consuming MJPEG and encoding x264 RTSP

alt definitions from /root/.remix.io/processes.json

{
  "processes": [
    {
      "name": "COI 02 ID 65",
      "description": "santoandre",
      "program": null,
      "args": [
        "-re",
        "-r",
        "15",
        "-i",
        "http://cameras.santoandre.sp.gov.br/coi02/ID_065",
        "-codec:v",
        "libx264",
        "-preset:v",
        "ultrafast",
        "-r",
        "15",
        "-g",
        "30",
        "-pix_fmt",
        "yuv420p",
        "-b:v",
        "500k",
        "-bufsize:v",
        "1000k",
        "-f",
        "flv",
        "rtmp://127.0.0.1/cam1"  **** serving rtmp to media mtx =D ***
      ],
      "cpus": "0-1"
    }
   ]
 }

[a comment by an automated account is not shown]
